ChameleonForms.Example/Controllers/ExampleFormsController.cs

```diff
@@ -103,30 +103,44 @@ public IFieldConfiguration ModifyConfig(IFieldConfiguration config)
         public string RequiredString { get; set; }
 
         public int RequiredInt { get; set; }
+

```

Comment:
FYI: There is a lot of whitespace changes in this file and some of the others (presumably the auto-format of ReSharper or Visual Studio). While I'm not adverse to having the format take place, in general I'd prefer to have it in a separate commit / PR so it's clear what the real change is.